treatment-service/integration-test/fetch_treatment_it_test.go

```diff
@@ -367,14 +367,17 @@ func (suite *TreatmentServiceTestSuite) SetupSuite() {
 	suite.managementServiceClient = managementClient
 	suite.managementServiceServer = managementServer
 
-	kafkaCluster := tu.NewKafkaCluster()
-	err = kafkaCluster.StartCluster()
+	composeFilePaths := []string{"docker-compose.yaml"}
+	kafka := testcontainers.NewLocalDockerCompose(composeFilePaths, strings.ToLower(uuid.New().String()))
```

Comment:
Can we explicitly specify the service name that should be started up - `kafka` in this case (something to the effect of `docker-compose up -d kafka`)? This is so that in case more services are added to the compose file, we can limit what should be started for each test. Or perhaps we can restructure the compose file to `docker-compose/kafka.yaml`. wdyt?